nipy/algorithms/graph/tests/test_forest.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
+
```

Comment:
On Thu, 03 Sep 2015, Matthew Brett wrote:

> In [1]nipy/algorithms/graph/tests/test_forest.py:

> > @@ -1,3 +1,5 @@
> > +from __future__ import print_function
> > +

> Are you asking if from __future__ import print_function is necessary? Yes, I
> believe so...

why?  

python2.7 -c 'print("from a function")'

works just fine.  Thanks for educating ;)